torch/csrc/api/include/torch/nn/options/linear.h

```diff
@@ -20,6 +20,17 @@ struct TORCH_API LinearOptions {
 
 // ============================================================================
 
+/// Options for the `Flatten` module.
+struct TORCH_API FlattenOptions {
+  FlattenOptions(int64_t start_dim, int64_t end_dim);
```

Comment:
I think for options that has two or more optional arguments, we usually don't provide the non-default constructor (e.g. `CosineEmbeddingLossOptions`), and we should likely remove the constructor here